### PR TITLE
Dev containers support for VSCode

### PR DESCRIPTION
**Pull request by xiru**
Add initial support for dev containers on VS Code.

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	// More info: https://containers.dev/implementors/json_reference/#port-attributes
+	"portsAttributes": {
+		"8080": {
+			"label": "Plone Backend",
+			"onAutoForward": "notify"
+		}
+	},
+
+	// Use 'postCreateCommand' to run commands after the container is created.
+	"postCreateCommand": "pip3 install --user -r requirements.txt"
```
Comment (davisagli) [inline .devcontainer/devcontainer.json]: @xiru We should run `make install` here. This takes care of some other steps that build the requirements file to include some packages from github (for the packages that have unreleased changes).

We'll need a base image that includes tools like make, git, and ssh.

**Review by davisagli — COMMENTED** [review]
@xiru Thanks for running with this while I was discussing docs with the GSOC students. Here are a couple things I think we'll need to update. Come grab me if you want to discuss.

**Comment by xiru** [comment]
@davisagli added a script to run the buildout as part of the dev container creation. Now we should be OK. Tested in my laptop and it worked as expected, environment was created, vc code sends a notification about the service running, you click it and opens a browset in you local machine, connecting localhost:8080 and it actually connects the 8080 port  inside the container.

**Comment by davisagli** [comment]
Closing in favor of #970, which uses `make install` and fixes the virtual hosting

**Pull request closed without merging**